assets/sass/legacy/_main.scss

```diff
@@ -58,21 +60,13 @@ table.src tr td.src {
   user-select: none;
 }
 
-/* FOCUS STATES - !important is needed to apply consistently to all elements */
-
-.js-focus-visible a.focus-visible,
-.js-focus-visible input.btn.focus-visible,
-.js-focus-visible button.focus-visible { /* Mostly for links in text */
-    background-color: #ffdd00 !important;
-    color: #333 !important;
-    outline: 3px solid rgba(0, 0, 0, 0) !important;
-    text-decoration: underline !important;
-}
+/* FALLBACK FOCUS STATES */
 
-.js-focus-visible input.focus-visible, 
-.js-focus-visible select.focus-visible, 
-.js-focus-visible div.focus-visible {
-    outline: 3px solid #ffdd00 !important;
+.js-focus-visible a,
+.js-focus-visible input,
+.js-focus-visible select,
+.js-focus-visible div {
+	@include focus-with-outline-only;
```

Comment:
Can this fallback be moved out of the legacy to new scss files? For example in it's own file to make clear that it's a fallback.

Reply:
Good point, it should definitely come out of the legacy file. 

I'd be inclined to leave the extra ones I've hooked into where they are (like `.navbar`, where the class didn't get any re-writing other than adding the mixin).

Reply:
Is the fallback needed anymore?

Reply:
Yes, but maybe fallback isn't the best description.

Maybe something like "generic focus style" would be a better description.

I was trying to say that it applied to components which don't have a specific focus style defined for the component - at present, this one is used for all form inputs, as we've not got our own components for those.

Reply:
> I'd be inclined to leave the extra ones I've hooked into where they are (like .navbar, where the class didn't get any re-writing other than adding the mixin).

I think that is fine. Not best, but fine.